components/molecules/HighlightInput/highlight-input-form.tsx

```diff
@@ -122,11 +122,12 @@ const HighlightInputForm = ({ refreshCallback }: HighlightInputFormProps): JSX.E
             <div className="flex pr-2">
               <input
                 value={title}
+                maxLength={49}
                 onChange={(e) => setTitle(e.target.value)}
                 className="flex-1 focus:outline-none"
                 type="text"
                 placeholder={
-                  isDivFocused ? "Add title (optional)" : "Click here to highlight your merged PRs and provide a link!"
+                  isDivFocused ? "Add title (optional, 49 char allowed)" : "Click here to highlight your merged PRs and provide a link!"
```

Comment:
They will get a message on the form already if they go over the limit. We should make the `maxLength` 50 characters to match the API

```suggestion
                  isDivFocused ? "Add title (optional)" : "Click here to highlight your merged PRs and provide a link!"
```

Reply:
Done, Thanks you for your comment.